tracer/src/Datadog.Trace/Agent/MessagePack/SpanMessagePackFormatter.cs

```diff
@@ -14,9 +14,11 @@
 
 namespace Datadog.Trace.Agent.MessagePack
 {
-    internal class SpanMessagePackFormatter : IMessagePackFormatter<Span>
+    internal class SpanMessagePackFormatter : IMessagePackFormatter<TraceChunkModel>,
+                                              IMessagePackFormatter<SpanModel>,
+                                              IMessagePackFormatter<Span> // IMessagePackFormatter<Span> is not allowed, use IMessagePackFormatter<SpanModel> instead
```

Comment:
Can you delete `IMessagePackFormatter<Span>`?

Reply:
I was trying to make sure nobody uses `SpanMessagePackFormatter` to serialize `Span`, but the check in `SpanFormatterResolver` might be enough. We never use `SpanMessagePackFormatter` directly.

Reply:
Done.